WaveformTimeline/Controls/Curtains.cs

```diff
@@ -409,12 +409,12 @@ private void CurtainMoving()
         private void MoveCuePoint(Point currentPoint, double leftCorner, double rightCorner)
         {
             Debug.Assert(_cueMarksCanvas != null);
-            Debug.Assert(_selectedCuePointLine != null);
+            // Debug.Assert(_selectedCuePointLine != null);
             Debug.Assert(_leftSideCurtain != null);
             Debug.Assert(_rightSideCurtain != null);
-            if (_selectedCuePointMark == null) return;
+            if (_selectedCuePointMark == null || _selectedCuePointLine != null) return;
```

Comment:
Um, no I meant there should be a null check for both of them and if either one is null, bail

Reply:
I'm sorry. I'm blind.

Reply:
Not to worry, happens to all of us! :)